backup.sh

```
@@ -1,8 +1,9 @@
 #!/bin/bash
-<< help
-This is a shell script to take backups 
-can also be used with cron
-help
+<< help1
```

Shellcheck (0.10.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)